### DISCUSSION TITLE
How to create a Swoole\Http\Response instance for unit testing?

### DISCUSSION THREAD

**Discussion by razonyang** [swoole/swoole-src#4831] [Q&A]
Hi, I would like to create a `Swoole\Http\Response` instance for unit testing, pseudo-code as following.

```php
$out = fopen('/tmp/response.out');
$response = createResponse($out); // Swoole\Http\Response
$response->write('OK');
$this->assertContains('OK', fread($out, $size));
```

I want to read the output of `Swoole\Http\Response`, is there any way to achieve that?

Thanks.

**Comment by henrywood** [comment]
It appears there is a static Response::create() method ?
